python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -161,7 +161,7 @@ class MatMul(OnnxOpConverter):
 
     @classmethod
     def _impl_v13(cls, bb, inputs, attr):
-        return relax.op.matmul(inputs[0], inputs[1])
+        return bb.emit(relax.op.matmul(inputs[0], inputs[1]))
```

Comment:
what does `bb.emit` do versus just calling `relax.op.matmul` directly?

Reply:
Good question! As far as I know, it generates another result variable that has a name hint, which prevents the later functions calls from breaking.

Reply:
With #36 in place probably we don't need this change. I'll rebase and update after it's merged.